e2e/load_test.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2021 Tetrate
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"bytes"
+	"log"
+	"testing"
+	"time"
+
+	"fortio.org/fortio/fhttp"
+	"fortio.org/fortio/fnet"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_http_load_with_high_bandwidth(t *testing.T) {
+	stdErr, kill := startEnvoyWith("network", t, 8001)
+	defer kill()
+
+	states := []struct {
+		numCalls    int64
```

Comment:
why numCall always set to 1?

Reply:
I mean this scenario is focusing on huge data per request, so I set it to 1 for reducing execution time.

Reply:
At first, I was going to split the QPS-based test from this PR, but I'm going to integrate a QPS-based scenario to this PR (e.g., the one with a 10000 QPS and 256KB payload) with a subtest.

Reply:
done